pyproject.toml

```diff
@@ -49,7 +47,7 @@ numpy = "<=1.26.4"
 isaura = {version="0.1", optional=true}
 pytest = {version = "^7.4.0", optional = true}
 fuzzywuzzy = {version = "^0.18.0", optional = true}
-sphinx = {version = ">=5.3.0", optional = true} # For compatibility with python 3.7.x
+sphinx = {version = ">=5.3.0", optional = true} 
```

Comment:
Maybe we actually check which is the minimum version we need for 3.8?

Reply:
ok sure